Corrected the XenoVents event

Colin-Tel opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->

Moved it out of the Entities folder and into the proper directory, added the missing locale file, and also lowered the weight of the event.

ShatteredSwords commented:
Would rerun, looks like something is wrong with the server config and not with the map

DebugOk reviewed (approved):
Tested it on the server, seems to be good

Colin-Tel committed:
Corrected XenoVents event

Moved it out of the Entities folder and into the proper directory, also lowered the weight of the event.

Colin-Tel committed:
Update xeno-vent.ftl

:trollface:
